src/templates/editor/review_revisions.html

```diff
@@ -1,91 +1,110 @@
+            class="btn btn-xs btn-task pull-right"><i
+            class="fa fa-plus">&nbsp;</i>Request Revisions</a></h2>
+    <table class="table table-bordered">
+        <tr>
+            <th>Revision</th>
+            <th>Requested</th>
+            <th>Due</th>
+        </tr>
+        {% for revision in revision_requests %}
+            {% ifequal revision.id revision_id %}
+                <tr class="selected-row-table">
+                    {% else %}
+                <tr>
+            {% endifequal %}
+        <td>Revision {{ forloop.counter }} </td>
+        <td>{{ revision.requested }}</td>
+        {% ifequal revision.id revision_id %}
+            <td class="selected-row-table-arrow"
+        {% else %}
+            <td
+        {% endifequal %} width="25%">{% if revision.completed %}
+            <a class="btn btn-task"
+               href="{% url 'editor_view_revisions' submission.id revision.id %}">View</a>
+        {% else %}
+            <small>{{ revision.due }} <a
+                    href="{% url 'editor_change_revision_due_date' submission.id revision.id %}"><i
+                    class="fa fa-edit">&nbsp;</i></a></small>{% endif %}</td>
+        </tr>
+        {% endfor %}
+    </table>
 {% endif %}
 {% if press.general.editorial_assignment_feature %}
     <hr>
     <h2>Editorial Review Assignments</h2>
-    <a href="{% url 'add_editorial_review' 'submission' submission.id %}" class="btn btn-sm btn-accept pull-right"
+    <a href="{% url 'add_editorial_review' 'submission' submission.id %}"
+       class="btn btn-sm btn-accept pull-right"
        style="margin-bottom:5px"><i class="fa fa-plus">&nbsp;</i>Create
     </a>
 
     <table class="table table-bordered">
         <tr>
-          <th>Editorial Reviewer</th>
-          <th>Info</th>
-          <th style="border-right-width: 1"></th>
-        </tr>
-    {% for review in editorial_review_assignments %}
-        <tr>
-        <td>{{ review.user.profile.full_name }}</td>
-        <td>
-            {% if review.completed %}
-            <font color="green">Completed: {{ review.completed }}</font>
-            {% else %}
-                {% if review.is_overdue %}
-                    <font color="red">Overdue: {{ review.due }}</font>
-                {% else %}
-                    Due: {{ review.due }}
-                {% endif %}
-            {% endif %}
-        </td>
-        <td width="25%"><a class="btn btn-task" href="{% url 'editor_view_editorial_review' submission.id review.id %}">View</a></td>
+            <th>Editorial Reviewer</th>
+            <th>Info</th>
+            <th style="border-right-width: 1"></th>
         </tr>
+        {% for review in editorial_review_assignments %}
+            <tr>
+                <td>{{ review.user.profile.full_name }}</td>
+                <td>
+                    {% if review.completed %}
+                        <font color="green">Completed: {{ review.completed }}</font>
```

Comment:
Apparently the `font` tag is deprecated I first started writing HTML about 10 years ago and have never seen one before.

Reply:
You're right, it's been replaced with `css`